hal/LowPowerTickerWrapper.cpp

```diff
@@ -271,13 +263,15 @@ void LowPowerTickerWrapper::_schedule_match(timestamp_t current)
         return;
     }
 
-    if (!too_close) {
+    if (!too_close || _suspended) {
```

Comment:
Rather than checking for `_suspended` here I recommend updating `LowPowerTickerWrapper::read` and `LowPowerTickerWrapper::set_interrupt` to check for `_suspended` can call straight into the real interface. That way it is clear in the code that when this is suspended everything is passed through directly.

If you make this change you'll also want to add a delay in `LowPowerTickerWrapper::resume` similar to the one in `LowPowerTickerWrapper::suspend` to ensure any ongoing hardware writes have taken effect.

Reply:
Done.